File: blog/src/main/java/cn/pompip/myblog/exe/ArticleWrapper.java
package cn.pompip.myblog.exe;

import cn.pompip.lib.entity.ArticleEntity;

import java.sql.Timestamp;
import java.util.Scanner;

public class ArticleWrapper {

    private String title;
    private String content;
    private Timestamp createTimestamp;

    public ArticleWrapper(String con) {
        Scanner scanner = new Scanner(con);
        String line = scanner.nextLine();
        if (line != null && line.startsWith("# ")) {
            this.title = line.substring(2);
        } else {
            title = "无标题";
        }

        content = con;
        createTimestamp = new Timestamp(System.currentTimeMillis());
    }

    public ArticleEntity createArticleEntity() {
        ArticleEntity a = new ArticleEntity();
        a.setContent(content);
        a.setTitle(title);
        a.setCreateTimestamp(createTimestamp);
        return a;

    }
}

File: src/main/kotlin/cn/pompip/myblog/exe/ArticleWrapper.kt
package cn.pompip.myblog.exe

import cn.pompip.myblog.entity.ArticleEntity
import java.sql.Timestamp
import java.util.*

class ArticleWrapper( con: String) {
    var title: String = ""
    var content: String
    var createTimestamp: Timestamp;

    init {
        val scanner = Scanner(con)
        val line = scanner.nextLine()
        if (line != null && line.startsWith("# ")) {
            this.title = line.substring(2)
        } else {
            title = "无标题"
        }

        content = con;
        createTimestamp = Timestamp(System.currentTimeMillis())
    }

    fun createArticleEntity(): ArticleEntity {
        val a = ArticleEntity();
        a.content = content;
        a.title = title;
        a.createTimestamp = createTimestamp;
        return a;

    }

}